### PR TITLE
Loans dg/ug

### PR DESCRIPTION
**Pull request by KartikeKhandelwal**

### CODE REVIEW

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -610,7 +610,10 @@ describes the behaviour of `DeleteTransactionCommand` at each step:
 
 === Repeaters
 
-// tag::repeateradd[]
```
Comment (weiyang13) [inline docs/DeveloperGuide.adoc]: Please revert the changes you have made here for the repeater section. The "commented" tags are necessary for creating PPP

**Review by weiyang13 — CHANGES_REQUESTED** [review]
Storage code looks fine, but not sure if there'll be problems.

Did not really read through your part of the documentation, but I noticed that you made changes to the repeaters section. Please revert those changes (end result: no change at all, no merge conflict possible for @LeeYiyuan ) before you merge this PR

File: docs/DeveloperGuide.adoc (hunk 1 above)
Comment (troomtroom) [reply docs/DeveloperGuide.adoc]: Aah, I'll change that

File: src/main/java/seedu/ichifund/logic/parser/loan/AddLoanCommandParser.java [hunk 2]
```diff
@@ -99,9 +99,9 @@ public AddLoanCommand parse(String args) throws ParseException {
             endmonth = ParserUtil.parseMonth(argMultimap.getValue(PREFIX_END_MONTH).get());
             endyear = ParserUtil.parseYear(argMultimap.getValue(PREFIX_END_YEAR).get());
         } else {
-            endday = new Day("30");
-            endmonth = new Month("12");
-            endyear = new Year("2025");
+            endday = new Day("11");
+            endmonth = new Month("11");
```
Comment (weiyang13) [inline src/main/java/seedu/ichifund/logic/parser/loan/AddLoanCommandParser.java]: Why are there hard coded values here?

**Review by weiyang13 — APPROVED** [review]
Looks okay.

File: src/main/java/seedu/ichifund/logic/parser/loan/AddLoanCommandParser.java (hunk 2 above)
Comment (troomtroom) [reply src/main/java/seedu/ichifund/logic/parser/loan/AddLoanCommandParser.java]: for fallback defaults.
